src/machine/machine_atmega328p.go

```diff
@@ -467,3 +468,96 @@ var SPI0 = SPI{
 	sdo:  PB3,
 	sdi:  PB4,
 	cs:   PB2}
+
+// Pin Change Interrupts
+var (
+	pinCallbacks [3][8]func(Pin)
+	interrupts   [3]*interrupt.Interrupt
+	previous     [3]uint8
+)
+
+func handlePCINTInterrupts(intr uint8, port *volatile.Register8) {
+	current := port.Get()
+	for i := uint8(0); i < 8; i++ {
+		if (current>>i)&0x01 == 0x01 && pinCallbacks[intr][i] != nil {
+			pinCallbacks[intr][i](Pin(i * (intr + 1)))
+		}
+	}
+	previous[intr] = current
+}
+
+func handlePCINT0Interrupts(intr interrupt.Interrupt) {
+	handlePCINTInterrupts(avr.IRQ_PCINT0-avr.IRQ_PCINT0, avr.PINB)
+}
+
+func handlePCINT1Interrupts(intr interrupt.Interrupt) {
+	handlePCINTInterrupts(avr.IRQ_PCINT1-avr.IRQ_PCINT0, avr.PINC)
+}
+
+func handlePCINT2Interrupts(intr interrupt.Interrupt) {
+	handlePCINTInterrupts(avr.IRQ_PCINT2-avr.IRQ_PCINT0, avr.PIND)
+}
+
+func SetInterrupt(pin Pin, callback func(Pin)) (err error) {
+	if callback == nil {
+		// Disable this pin interrupt
+		return errors.New("Not implemented")
+	} else {
+		var PCMSK_REG *volatile.Register8
+		var PCIE uint8
+		var PCINT uint8
+		var IRQ uint8
+		var PIN0 Pin
+		//var PIN *volatile.Register8
```

Comment:
Here is some more unused code.

Reply:
Fixed it. What's the convention for "Resolve conversation" here? 
I've worked on projects where the person who opens it should resolve and ones where the person who makes the fix resolves it...

Reply:
We don't really have a convention here. But if a comment is addressed I would consider that resolved (either by fixing it as requested or explaining why it's the way it is - there may be a good reason the reviewer didn't see!)